modules/nw-ovn-kubernetes-migration-about-live.adoc

```diff
@@ -0,0 +1,166 @@
+
+* Continuous service availability
+* Minimized downtime
+* Automatic node rebooting
+* Seamless transition from the OpenShift SDN network plugin to the OVN-Kubernetes network plugin. 
+
+The live migration method is valuable for deployment types that require constant service availability, such as Azure Red Hat OpenShift (ARO), Red Hat OpenShift Dedicated, or Red Hat OpenShift Service on AWS. It is not available for HyperShift deployment types. 
+
+== Considerations for migrating to the OVN-Kubernetes network plugin
+
+* When the live migration procedure begins, multitenant isolation is permanently disabled. Multitenant isolation is only supported on OpenShift SDN. 
+
+* During the live migration, when the cluster is running both OVN-Kubernetes and OpenShift SDN, multicast and egress IP are temporarily disabled for both CNIs.
+
+* Although a rollback procedure is provided, the migration is intended to be a one-way process. For users that want to rollback to OpenShift-SDN, migration from OpenShift-SDN to OVN-Kubernetes must have succeeded. Users can follow the same procedure below to migrate to the OpenShift SDN network plugin from the OVN-Kubernetes network plugin.
+
+* The live migration is only supported on managed clusters. It is not supported on HyperShift clusters. 
+
+== Information about the OpenShift SDN live migration script
+
+The live migration method for migrating from OpenShift SDN network plugin to the OVN-Kubernetes network plugin is a two-step procedure that contains multiple internal processes that are executed by a wrapper script. When running the two commands, the following events occur:
+
+. The Cluster Network Operator (CNO) sets the `NetworkTypeMigrationInProgress` condition to `True` in the `status` field of the `network.config` custom resource. If there are any unsupported features enabled in the cluster, the CNO sets the `NetworkTypeMigrationInProgress` to `False` and alerts administrators that the migration will not start. Users must disable these unsupported features before starting the live migration. 
+
+. The Cluster Network Operator checks the current hardware MTU and the cluster MTU to determine a routable MTU during migration. These changes trigger the Machine Config Operator to apply a new machine configuration to each machine config pool and reboot all nodes:
+
+** If the current hardware MTU is `1500`, and the cluster MTU is not specified, the CNO sets the `.spec.migration.mtu` of the `network.operator` CR with the MTU of OVN-Kubernetes: 
+*** `.spec.migration.mtu.network.from` is reduced from `1450` to `1400`
+*** `.spec.migration.mtu.machine.from` remains `1500`
+
+** If the current hardware MTU is `9000`, and the cluster MTU is specified explicitly to `1250` for OpenShift SDN, the CNO sets the `.spec.migration.mtu` of the `network.operator` CR with the MTU of OVN-Kubernetes:
+*** `.spec.migration.mtu.network.from` is reduced from `1250` to `1200`
+*** `.spec.migration.mtu.network.from` stays `9000`
```

Comment:
These are just examples. Users shall be aware that the cluster MTU will be reduced by 50 bytes after the migration.

Reply:
Changed to: 

1. The Cluster Network Operator checks the current hardware MTU and the cluster MTU to determine a routable MTU during migration. These changes trigger the Machine Config Operator to apply a new machine configuration to each machine config pool and reboot all nodes. After migration, MTU values are reduced by 50 bytes. For example:

     - If the current hardware MTU is `1500`, and the cluster MTU is not specified, the CNO sets the `.spec.migration.mtu` of the `network.operator` CR with the MTU of OVN-Kubernetes: 
         - `.spec.migration.mtu.network.from` is reduced from `1450` to `1400`
         - `.spec.migration.mtu.machine.from` remains `1500`
         
  Which mentions that they're reduced by 50 and provides an example. I can remove the example if suggested.